src/lib.rs

```diff
@@ -1,3 +1,4 @@
+// t1
```

Comment:
This line should go, I forgot to remove it.

Reply:
Agreed.  Will be resolved in the next push.